processor/configuration/src/main/java/com/redhat/service/smartevents/processor/AbstractGatewayConnector.java

```diff
@@ -59,9 +63,19 @@ public JsonNode connectorPayload(T gateway, String topicName) {
 
         addConnectorSpecificPayload(gateway, topicName, definition);
 
+        addErrorHandlerPayload(errorHandlerTopicName, definition);
+
         return definition;
     }
 
+    protected void addErrorHandlerPayload(String errorHandlerTopicName, ObjectNode definition) {
```

Comment:
Configure MC's `error_handler` to return failures to a DLQ (our `error-handler` topic/back channel).

We could have a DLQ per `Processor` that intercepts the response from MC and wraps it into an error message that is forwarded to the Sidecar however at the moment the Sidecar has the burden of receiving different message types and handling them accordingly [*]. The payload is currently simply the message we sent to MC (that said MC doesn't return any notion of _why_ it failed).

[*] If the message is not a formatted Error message (to be added by me soon, when I add some handling for Filters/Transformations/Send to Webhook Actions) it can be inferred that it's a MC failure... otherwise we need lots more topics (a back channel per `Processor` and a back channel per `Bridge` so things start to get crazy.. and are we resource limited on Kafka topics!?!)